### ISSUE TITLE
A dedicated Tab or E-Mail for results of finished tournaments

### ISSUE THREAD

**Issue by enei200** [lemonpole/liga-public#82]
I feel like it would be a great QOL change, as there is really no way to see the results of the tournament/qualifiers after you finish your last match there. I think either a dedicated tab for any ongoing/finished tournaments (which I would feel would be great for immersion, as its always better to see that the world plays out with or without you) or an e-mail for a tournament you just finished with the finished bracket. First option is a lot to ask for, but maybe it's a good idea for a distant future? The e-mail though, I think, would be a good implementation for now.

**Comment by lemonpole** [comment]
Great idea. I've got a story filed in the app's private tracker to implement a dedicated tournaments page. 

I'm going to draw heavy inspiration from Football Manager's league pages. I even installed FM24 to be able to get some UI/UX idea 😅

![image](https://github.com/user-attachments/assets/93c41ba5-6068-425f-a5be-9bc23873d798)

In terms of a short term solution I think the Match Results page might be what you are looking for. It allows you to view the results for specific tournaments.

![image](https://github.com/user-attachments/assets/8d4b4f71-24ea-407a-8b03-8fc8f2462ae2)

**Comment by enei200** [comment]
Thanks for the response! I know about the match results page, but it's a little hard to follow through, when it's a Round-Robin/Division tournament. But that's not a big deal. Thanks again. I'm staying tuned for the updates :D

**Comment by lemonpole** [comment]
By the way @enei200 I made a discord server to streamline the feedback loop. It's been mentioned by others that it'd be a good platform to report bugs, request features, etc.

If you're interested:

https://discord.gg/ZaEwHfDD5N

**Comment by lemonpole** [comment]
Hey, just letting you know this is finally in the works 💪

You'll be able to look at all the past seasons of every competition and also see who won, final standings/brackets, etc.

![image](https://github.com/user-attachments/assets/dd915660-4ee6-4f90-9cab-32a5813acb74)

**Comment by lemonpole** [comment]
This was shipped with Beta 14. Give it a try and please let me know what you think!
